### DISCUSSION TITLE
Add multiple payment gateway

### DISCUSSION THREAD

**Discussion by gunasekharravilla** [hoc081098/Movie-Ticket-Booking#50] [Ideas]
it gives many payment options  like net banking, debit card, wallets, UPI(ex payment gateways: Instamojo, razor pay, payu.in)

please include any of these

**Comment by hoc081098** [comment]
I will consider. Thanks 😃

**Reply by gunasekharravilla** [reply]
😄😄Thank you, Actually I'm a btech student just learning  all these 

I'm suggesting all this because how the realtime process is going  and we have to develop more than that to create  good product
